src/Npgsql/NpgsqlRawCopyStream.cs

```diff
@@ -174,9 +174,16 @@ async Task<int> Read(byte[] buffer, int offset, int count, bool async)
 
             if (_leftToReadInDataMsg == 0)
             {
-                // We've consumed the current DataMessage (or haven't yet received the first),
-                // read the next message
-                var msg = await _connector.ReadMessage(async);
+                IBackendMessage msg;
+                try {
+                    // We've consumed the current DataMessage (or haven't yet received the first),
+                    // read the next message
+                    msg = await _connector.ReadMessage(async);
+                } catch {
```

Comment:
The only one issue here is coding style. Please use Allman style here (i.e. put braces on the next line here and for `try`).

Reply:
Fixed.